Marlin/language_tr.h

```diff
@@ -59,11 +62,13 @@
 #define MSG_PREHEAT_1                       _UxGT("Ön Isınma PLA")                                      // Ön Isınma PLA
 #define MSG_PREHEAT_1_N                     MSG_PREHEAT_1 _UxGT(" ")                                    //
 #define MSG_PREHEAT_1_ALL                   MSG_PREHEAT_1 _UxGT(" Tüm")                                 //  Tüm
+#define MSG_PREHEAT_1_END                   MSG_PREHEAT_1 _UxGT(" Nozül")                               //  Nozül
```

Comment:
MSG_PREHEAT_1_END and MSG_PREHEAT_2_END does not end anything. When the related functions with these messages are chosen, the machine heats the hotend only. So the current translation is meaningful. There is no exact short word for Hotend in Turkish.